complete.py

```diff
@@ -108,6 +110,21 @@ def get_rectangles(filename):
     # Return the list of Rectangle objects
     return rectangle_objects
 
+def get_triangles(filename):
+    """
+    Read in from the YAML file supplied and create
+    a list of triangles based on the input data.
+    """
+    with open(filename, 'r') as handle:
+        try:
+            data = yaml.load(handle)
+        except yaml.YAMLError as error:
+            print(error)
+            raise
+
+    triangle_objects = [Triangle(side['a'], side['b'], side['c']) for side in data]
```

Comment:
Another alternative:
`triangle_objects = [Triangle(item) for item in data]`

Then in the Triangle constructor, just break the dict apart.